Modules/TPC/include/TPC/PID.h

```diff
@@ -52,6 +55,8 @@ class PID final : public TaskInterface
 
  private:
   o2::tpc::qc::PID mQCPID{};
+  TProfile* pSeparationPower{};
```

Comment:
```suggestion
  TProfile* mSeparationPower{};
```
see also comments below.